src/OAM_DMA_Transfer.md

```diff
@@ -13,11 +13,10 @@ Destination: $FE00-$FE9F
 ```
 
 The transfer takes 160 machine cycles: 152 microseconds in normal speed
-or 76 microseconds in CGB Double Speed Mode. On DMG, during this time,
-the CPU can access only HRAM (memory at $FF80-$FFFE); on CGB, the bus used
-by the source area cannot be used (this isn't understood well at the
-moment; it's recommended to assume same behavior as DMG). For this
-reason, the programmer must copy a short procedure into HRAM, and use
+or 76 microseconds in CGB Double Speed Mode.
+This is much faster than a CPU-driven copy.
+On DMG, during this time, the CPU can access only HRAM (memory at $FF80-$FFFE).
```

Comment:
Do you mean put everything related to the bus conflict below the heading and everything else above the heading?

Reply:
In my opinion, the routines must come after the bus conflict, otherwise it is not obvious that they should, or why they should be copied to HRAM. Two headings may work better, then.

Reply:
Adding section "Best practices"